macros/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+use proc_macro2::TokenStream;
+
+mod pro;
```

Comment:
feature gate this?

Reply:
[d9966ff](https://github.com/geo-engine/geoengine/pull/859/commits/d9966ff02c9088611077b05e0b218c477ec6f378)